src/cmd/cli/command/commands.go

```diff
@@ -176,8 +175,10 @@ func SetupCommands(version string) {
 	RootCmd.AddCommand(newCmd)
 
 	// Get Services Command
-	getServicesCmd.Flags().BoolP("long", "l", false, "show more details")
-	RootCmd.AddCommand(getServicesCmd)
+	lsCommand := makeComposeLsCmd()
+	lsCommand.Use = "services"
```

Comment:
why wouldn't we set `Use` and `Aliases` inside `makeComposeLsCmd`? Shouldn't they be the same no matter where we add the cmd?